Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon_description.md

```diff
@@ -1,11 +1,13 @@
-To get an The API client ID and secret, contact the crowdstrike support: support@crowdstrike.com
+To get an API client ID and secret, contact [CrowdStrike support](mailto:support@crowdstrike.com).
 
 
 #### Important:
-This integration is enabled by default for the new CrowdStrike Raptor version. For the older API version (pre-Raptor release), ensure you check the "Use legacy API" checkbox and select the Legacy mapper as well.
 
-### Required API client scope
-In order to use the CrowdStrike Falcon integration, your API client must be provisioned with the following scope and permissions:
+This integration is enabled by default for the new CrowdStrike Raptor version. <~XSOAR>For the older API version (pre-Raptor release), check the "Use legacy API" checkbox and select the Legacy mapper as well.</~XSOAR>
```

Comment:
Have we verified this looks as expected in the product and in the documentation?

Reply:
Yes, I can show you an in person demo if needed